human_info/src/run.py

```diff
@@ -114,57 +82,105 @@ def add_centroids(frame, center, avg):
     return frame
 
 
-def process_frame(frame):
-    """
-    Calculatest he distance and position of the player and the detected human features
 
-    @param frame The camera frame
-    @return (distance, (frame center, pose average, horizontal_angle, vertical_angle), poses)
-    """
-
-    # prediction
-    poses, inference_time = ENGINE.DetectPosesInImage(np.uint8(frame))
-
-    distance = cal_distance(poses)
-    position = pos_from_center(poses, frame.shape[:-1])
-    return distance, position, poses
 
 
 class Processor:
     def __init__(self):
         self.pub = rospy.Publisher('human_info', HumanPos, queue_size=1)
+        # we create the outlier_rejection here, because we need to continually save the distance to each feature, so this can not happen within one frame, but over the course of all the frames
+        self.outlier_rejection = filter.Outlier_Rejection()
+        self.BRIDGE = CvBridge()
+        self.average_filter = filter.Average_Filter()
+        rospack = rospkg.RosPack()
+        with open(rospack.get_path("phx_launch") + "/../config/software_config.yaml") as f:
+            sw_config = yaml.load(f, Loader=yaml.SafeLoader)
+        self.keypoint_filter = filter.Keypoint_Filter(sw_config["filter_constant_keypoints"], sw_config["num_of_frames_conitunuity"], FEATURES, sw_config["outlier_threshold_keypoint_pixels"])
```

Comment:
Little bit too long.
```suggestion
        self.keypoint_filter = filter.Keypoint_Filter(sw_config["filter_constant_keypoints"], 
            sw_config["num_of_frames_conitunuity"], FEATURES, sw_config["outlier_threshold_keypoint_pixels"])
```

Reply:
True. I made a new pep8-compliant version. Does that look acceptable?

Reply:
For me it still looks the same but I assume this is something with github.